Basic-Car-Maintenance/Shared/Settings/Views/SettingsView.swift

```diff
@@ -11,6 +11,7 @@ import UniformTypeIdentifiers
 import TipKit
 
 struct SettingsView: View {
+    
```

Comment:
be careful about adding extra space in files that you don't really change, if there's many lines like this, it makes the PR hard to review

and if you see it in lots of places, make a new issue and mention what files
I fixed this